app/src/test/java/com/example/mybeats/data/repository/ProductsRepositoryTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.mybeats.data.repository
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.example.mybeats.data.remote.api.ProductsApi
+import com.example.mybeats.data.remote.extension.toModel
+import com.example.mybeats.data.remote.model.ProductsBody
+import com.example.mybeats.data.remote.responses.ResultRemote
+import com.google.common.truth.Truth.assertThat
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.impl.annotations.MockK
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+@ExperimentalCoroutinesApi
+class ProductsRepositoryTest {
+
+    @get:Rule
+    var rule: TestRule = InstantTaskExecutorRule()
+
+    @MockK
+    private lateinit var productsApi: ProductsApi
+
+    @MockK
+    private lateinit var productsBody: ProductsBody
```

Comment:
The variable was never used, remember to remove import too

```suggestion
```

Reply:
Sure... I'll remove it.